src/test/java/org/datadog/jmxfetch/TestCommon.java

```diff
@@ -0,0 +1,257 @@
+package org.datadog.jmxfetch;
+
+import org.datadog.jmxfetch.reporter.ConsoleReporter;
+import org.datadog.jmxfetch.reporter.Reporter;
+import org.datadog.jmxfetch.util.CustomLogger;
+
+import java.lang.management.ManagementFactory;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Set;
+
+import javax.management.InstanceAlreadyExistsException;
+import javax.management.InstanceNotFoundException;
+import javax.management.MalformedObjectNameException;
+import javax.management.MBeanRegistrationException;
+import javax.management.MBeanServer;
+import javax.management.NotCompliantMBeanException;
+import javax.management.ObjectName;
+import org.junit.After;
+import org.junit.BeforeClass;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+import com.beust.jcommander.JCommander;
+import org.apache.log4j.Level;
+
+
+public class TestCommon {
```

Comment:
Could you check the indentation in this file? It seems to be using mostly tabs, but we're using spaces in all the other files.